templates/default/entity/User/profile/pronouns.tpl.php

```diff
@@ -0,0 +1,11 @@
+<p class="p-pronoun pronoun small">
```

Comment:
sorry to be a microformats prescriptivist, but I don't think p-pronoun is a recognized/documented property for [h-card](http://microformats.org/wiki/h-card#Properties).